### DISCUSSION TITLE
Angular + Vite?

### DISCUSSION THREAD

**Discussion by smlimasu** [storybookjs/storybook#29385] [Documentation]
I saw the storybook blog post about angular + vite instead of webpack builder integration and the guide on analogjs. I am not using analog for my project.

On the storybook website, under angular framework, there is no mention of angular and vite. 

My question is, what is the roadmap for using angular + vite? Are you guys planning to integrate it sometimes without the analog plugin? And how does using vite work at the moment, because the analog plugin did not work for me either. Thank you!

**Comment by valentinpalkovic** [comment]
Hi @smlimasu 

Thank you for your question!

We already [spiked](https://github.com/storybookjs/storybook/pull/26132) esbuild/vite support for Storybook and Angular, and it turned out that supporting esbuild/vite in Storybook for Angular isn't trivial. The Analogjs vite plugin has phenomenal support and is a significant first step in playing around with it. It doesn't mean you must use AnalogJS in your project. The plugin also works for pure Angular projects. Try it out and give feedback. We may add a `@storybook/angular-vite` framework, where the plugin is built in with further optimizations.
